src/containerTest/kotlin/com/quipalup/katydid/CreateMealLogEntryCT.kt

```diff
@@ -0,0 +1,73 @@
+package com.quipalup.katydid
+
+import io.restassured.RestAssured
+import io.restassured.module.kotlin.extensions.Extract
+import io.restassured.module.kotlin.extensions.Given
+import io.restassured.module.kotlin.extensions.Then
+import io.restassured.module.kotlin.extensions.When
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.web.server.LocalServerPort
+
+@SpringBootTest(
+    classes = [Katydid::class],
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT
+)
+class CreateMealLogEntryCT {
+
+    @LocalServerPort
+    private val port: Int = 0
+
+    @BeforeEach
+    fun setUp() {
+        RestAssured.port = port
+        RestAssured.enableLoggingOfRequestAndResponseIfValidationFails()
+    }
+
+    @Test
+    fun `create meal log entry`() {
+        Given {
+// TODO        contentType("application/vnd.api+json")
+            body(requestBody)
+        } When {
```

Comment:
```suggestion
    fun `create meal log entry`() {
        Given {
            contentType("application/vnd.api+json")
            body(requestBody)
        } When {
```

If you pay attention to the current github run (https://github.com/code-sherpas/katydid-service/pull/13/checks?check_run_id=2422741891), you'll see that the container test (CT) fails. If you run the CT in local you'll see that is failing because of a 415 Unsupported Media Type. So if you uncomment this line, this error is not happening anymore.